microlith/loki/alerting/couchbase/couchbase-rules.yaml

```
@@ -181,3 +181,22 @@ groups:
           title: Analytics Crash ({{ $labels.hostname }})
           description: "Analytics on node: {{ $labels.hostname }} has crashed."
           remediation: Review analytics_debug.log to identify the cause, or contact Couchbase Technical Support.
+
+      - alert: CB90052-malformedvBuckets
+        expr: |
+          count_over_time({file="memcached.log"} |~ "malformed data in file" |json message="message" | line_format "{{.message}}" | pattern "<_>WARNING (<m_bucket>)<_>malformed data in file<_>, <vbucket>" [1h]) > 0
+        for: 1m
+        labels:
+          job: couchbase_fluent_bit
+          kind: bucket
+          severity: warning
+          health_check_id: CB90052
+          health_check_name: malformedvBuckets
+          cluster: '{{ $labels.cluster }}'
+          node: '{{ $labels.hostname }}'
+          m_bucket: '{{ $labels.m_bucket}}' # when using K8s there is a `bucket` label but this wont be available to all users, hence this label
```

This will need to be called `bucket` for it to show up correctly in the dashboards. I think you can do
```suggestion
          bucket: '{{ $labels.m_bucket}}' # when using K8s there is a `bucket` label but this wont be available to all users, hence this label
```